src/styles/fonts.ts

```diff
@@ -1,3 +1,3 @@
-import { Open_Sans as OpenSans } from 'next/font/google';
+import { Inter } from 'next/font/google';
 
-export const openSans = OpenSans({ subsets: ['latin'] });
+export const sans = Inter({ subsets: ['latin'] });
```

Comment:
are we planning to keep the Inter font named as sans?

Reply:
yep! from jinkang:

it's better practice to name the variable according to its style (i.e. sans, sans-serif, display) just so it saves us the many file refactorings later (although we probably won't be the ones benefitting from it)